Formula/libgr.rb

```diff
@@ -0,0 +1,38 @@
+class Libgr < Formula
+  desc "GR framework: a graphics library for visualisation applications"
+  homepage "https://gr-framework.org/"
+  url "https://github.com/sciapp/gr/archive/v0.37.0.tar.gz"
+  sha256 "9120c4215cd2c10239f45f4cbf79dcb199a9e7e3bbbf36e5ce8f4e5eb2e70e24"
+
+  depends_on :xcode => :build
+  depends_on "cairo"
+
+  def install
+    system "make", "self", "GRDIR=#{prefix}"
+    system "make", "clean"
```

Comment:
Why is `make clean` necessary?

Reply:
If HomeBrew does cleanup the temporary directories during the build, then this is not necessary. I will try to look for the current behavior and remove if this is true

Reply:
It's not necessary. Removed.